### DISCUSSION TITLE
Official Integration with Google Drive

### DISCUSSION THREAD

**Discussion by ipcam4g** [directus/directus#16124] [Feature Requests]
Please release official Google Drive integration

**Comment by azrikahar** [comment]
Ref https://github.com/directus/directus/discussions/7588#discussioncomment-3943296 (just linking this comment for bi-directional reference)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
